tests/Tests/800__culler/culler.robot

```diff
@@ -183,14 +195,19 @@ Set Notebook Culler Timeout
     [Arguments]    ${new_timeout}
     ${hours}  ${minutes} =  Convert To Hours And Minutes  ${new_timeout}
     Sleep  5
-    ${disabled_field} =  Run Keyword And Return Status  Page Should Contain Element
+    ${disabled_field} =  Run Keyword And Return Status    Page Should Contain Element
     ...    xpath://input[@id="hour-input"][@disabled=""]
     IF  ${disabled_field}==True
         Click Element  xpath://input[@id="culler-timeout-limited"]
     END
     Input Text  //input[@id="hour-input"]  ${hours}
     Input Text  //input[@id="minute-input"]  ${minutes}
-    Save Changes In Cluster Settings
+    Sleep  0.5s
+    ${changed_setting} =  Run Keyword And Return Status    Page Should Contain Element
+    ...    xpath://button[.="Save changes"][@aria-disabled="false"]
+    IF  ${changed_setting}==True
+        Save Changes In Cluster Settings
+    END
 
 Disable Notebook Culler
```

Comment:
Could be in ODS.robot

Reply:
(This goes for all similar comments)
Why would other tests touch culler settings and configs? IMHO it makes more sense to centralize all culler-related keywords and tests in one place